packages/components/OmniLink.tsx

```diff
@@ -0,0 +1,61 @@
+import { useLinkProps } from "@react-navigation/native";
+import React, { ReactNode } from "react";
+import {
+  Platform,
+  StyleProp,
+  TouchableOpacity,
+  TouchableOpacityProps,
+  View,
+  ViewStyle,
+} from "react-native";
+
+export interface OmniLinkToType {
+  screen: string;
+  params?: object;
+}
+
+export const OmniLink: React.FC<{
+  to: OmniLinkToType;
+  action?: any | undefined;
+  children: ReactNode | undefined;
+  style?: StyleProp<ViewStyle | TouchableOpacityProps>;
+
+  disabled?: boolean;
+}> = ({ to, action, children, style, disabled, ...rest }) => {
```

Comment:
Basically ...rest should be [] here so it does not make sense to have it in args

Reply:
Thanks. I was just thinking how to make it as open as possible.
The solution is to delete rest altogether?